modules/ROOT/pages/quick-start.adoc

```diff
@@ -230,16 +231,18 @@ Using Maven command line
 ----
 # Parameters 'bonita.url', 'bonita.tech.user' and 'bonita.tech.password' are optionals. 
 # Use them if your runtime runs on an other URL than localhost:8080/bonita, 
-# or with technical user credentials different than install / install. 
+# or with technical user credentials different from install / install. 
 
 mvn verify [-Dbonita.url=<TARGET_RUNTIME_URL> -Dbonita.tech.user=<TECHNICAL_USER_USERNAME> -Dbonita.tech.password=<TECHNICAL_USER_PASSWORD>]
 ----
 
 ==== Advanced usage
 
-The test toolkit communicate with the target Bonita Runtime using HTTP. 
+The test toolkit communicates with the target Bonita Runtime using HTTP. 
 It is possible to fine tune some HTTP specific options using the following arguments:
 
 * `-Dbonita.client.disable.certificate.check=true|false` Default is `false`. Disable SSL certificate check when connecting to remote runtime using HTTPS.
 * `-Dbonita.client.http.connection.timeout=10` Default is `10`. Timeout in seconds used for the HTTP connection timeout.
 
+=== To Go further
```

Comment:
Please add a line break after a title for lisibility (no impact on the generated HTML, just the source code).
```suggestion
=== To Go further

```